website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
UI users are _already_ being checked against a 500 character limit on the characters visible to them, no? I'm proposing that we implement the same check on the server-side, which would require a little more documentation for API users ("500 characters after converting to Markdown and stripping HTML"). No?